card/mod/05_standard/set/type/signup.rb

```diff
@@ -14,7 +14,7 @@ def default_new_args args
       }
     )
 
-    if Auth.signed_in? and args[:account].confirm_ok?
+    if Auth.signed_in? && args[:account].confirm_ok?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.